recipes/tex-fmt/recipe.yaml

```
@@ -0,0 +1,42 @@
+context:
+  version: "0.5.2"
+
+package:
+  name: tex-fmt
+  version: ${{ version }}
+
+source:
+  url: https://github.com/WGUNDERWOOD/tex-fmt/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: e9dd89236781ea6f781b00d9452a071311cebd175424baea08bc03e8863f314c
+
+build:
+  number: 0
```

```suggestion
  number: 0
  script_env:
    - CARGO_PROFILE_RELEASE_STRIP=symbols
    - CARGO_PROFILE_RELEASE_LTO=thin
```
We also want these environment variables set to strip unneeded symbols and run link time optimization.